src/test/java/org/springframework/integration/aws/inbound/KinesisMessageDrivenChannelAdapterTests.java

```diff
@@ -236,16 +236,17 @@ void testKinesisMessageDrivenChannelAdapter() {
 		assertThat(message.getPayload()).isInstanceOf(List.class);
 		messagePayload = (List<String>) message.getPayload();
 		assertThat(messagePayload).size().isEqualTo(2);
```

Comment:
Not sure if the change was automated, but I believe AssertJ recipe is like this `assertThat(messagePayload).hasSize(2);`.
Or Moderne recipe can be improved, or I miss something if this has not been fixed.

Reply:
Good one, this is not supported yet by OpenRewrite / Moderne. We have a rule for it, but it's not available via the platform yet. 

For now, manually applied the change.

CC: @timtebeek